datanode/space_manager.go

```diff
@@ -101,6 +109,56 @@ func (manager *SpaceManager) Stop() {
 	wg.Wait()
 }
 
+func (manager *SpaceManager) StartDiskSample() {
+	manager.samplerDone = make(chan struct{})
+	go func() {
+		for {
+			select {
+			case <-manager.samplerDone:
+				return
+			default:
+				var partitions []*disk.PartitionStat
+				func() {
+					manager.diskMutex.RLock()
+					defer manager.diskMutex.RUnlock()
```

Comment:
only keep disk util sample

Reply:
this is disk partition information, we need it to get disk util information.